app/models/user.rb

```diff
@@ -6,4 +6,8 @@ class User < ApplicationRecord
   validates :photo, presence: true, length: { minimum: MIN_LENGTH }
   validates :bio, presence: true, length: { minimum: MIN_LENGTH }
   has_many :posts, foreign_key: :author_id
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] Remember a user can have many `comments` and `likes` on a post please add associations
```ruby
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
You are right